content/en/docs/guides/metadata.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Metadata
+description: >-
+  Explains what metadata is, how it is transmitted, and what it is used for.
+---
+
+### Overview
+
+Metadata is a side channel that allows clients and servers to provide
+information to each other that is associated with the data messages.
+
+gRPC metadata is a key-value pair of data that is sent with a gRPC request or
+response. It is used to provide additional information about the call, such as
+authentication credentials, tracing information, or custom headers.
+
+gRPC metadata is implemented using HTTP/2 headers. The keys and values are ASCII
```

Comment:
The values are ASCII strings but can be binary data reads weird. I see the similar sentence in the gRPC concepts, but it reads better there (albeit, not being clear that only values may be binary).

Reply:
reworded